### PR TITLE
Fix for Issue #131: Extract course title from Class Schedule Listings page and Details Course Info page.

### PR DESCRIPTION
**Pull request by NishchintDhawan**
Fix for Issue #131: Extract course title from Class Schedule Listings page and Details Course Info page.

### CODE REVIEW

File: src/pages/detailedClassInformation/index.ts [hunk 1]
```diff
@@ -21,15 +21,24 @@ export const detailedClassInfoExtractor = ($: cheerio.Root): DetailedClassInform
...
+
+  const titleInfo = titleRegex.exec(titleElement.text())![1].trim();
+
   // initialize data to return
-  const data: DetailedClassInformation = { ...transformSeating(seatInfo) };
+  const data: DetailedClassInformation = {
+    ... {
+      title: titleInfo
+    }, ...transformSeating(seatInfo)
+  };
```
Comment (aomi) [inline src/pages/detailedClassInformation/index.ts]: I'm curious to why you're spreading the first object? Could you just not do `title: titleInfo` in the first object block?

**Review by aomi — COMMENTED** [review]
Clean and simple :100:

File: src/pages/detailedClassInformation/index.ts (hunk 1 above)
Comment (NishchintDhawan) [reply src/pages/detailedClassInformation/index.ts]: Yes that makes sense. I saw that previously it was written as `…transformSeating(seatInfo)` so I tried to keep that spread operator.

Comment (aomi) [reply src/pages/detailedClassInformation/index.ts]: @NishchintDhawan can you address this part as this is the only thing blocking an approval :+1:

**Review by aomi — APPROVED** [review]

**Pull request merged**